tox.ini

```diff
@@ -25,16 +25,18 @@ commands = python-codacy-coverage -r coverage.xml
 [testenv:publish]
 changedir =
 passenv =
-  TWINE_*
+    TWINE_*
+    CI_COMMIT_*
 skip_install=true
 deps =
-    setuptools_scm
+    setuptools_scm==8.0.4
```

Comment:
do we really need to pin setuptools_scm?
If so, maybe we should pin it in pyproject.toml instead?

Reply:
already in there, second line.